nominal/cli/util/global_decorators.py

```diff
@@ -137,10 +137,17 @@ def wrapped_function(
         trust_store_path: pathlib.Path | None,
         **kwargs: Param.kwargs,
     ) -> T:
-        api_token = get_token(base_url, token_path.expanduser().resolve()) if token is None else token
+        if token is None:
+            if token_path.exists():
+                token = get_token(base_url, token_path)
+            else:
+                raise ValueError(
+                    f"Cannot instantiate client: no token provided and token path {token_path} does not exist."
+                )
+
         client = NominalClient.create(
             base_url,
-            api_token,
+            token=token,
```

Comment:
Is it ok for this to become a named parameter?

Reply:
Yeah I was only worried about issues with positional vs named causing issues. Sounds good to me.